lighthouse-cli/test/smokehouse/perf/expectations.js

```diff
@@ -94,19 +94,19 @@ module.exports = [
             items: [
               {
                 resourceType: 'total',
-                countOverBudget: '3 requests',
-                sizeOverBudget: '65000±1000',
+                countOverBudget: '4 requests',
+                sizeOverBudget: '113000±1000',
+              },
+              {
+                resourceType: 'font',
+                countOverBudget: '1 request',
+                sizeOverBudget: '52000±1000',
               },
               {
                 resourceType: 'script',
                 countOverBudget: '2 requests',
                 sizeOverBudget: '25000±1000',
               },
-              {
-                resourceType: 'font',
-                countOverBudget: undefined,
```

Comment:
the intention was to make sure we got a good mixture of the combinations of defined and undefined for these entries, so ideally this wouldn't change from `undefined`.

Maybe the "Undefined items" comment above could be changed to be a little more strongly worded to warn about that and we could change `corsfont` to something completely different so it doesn't look like a bug that it's not getting the font?

Reply:
> change corsfont to something completely different so it doesn't look like a bug that it's not getting the font?

yea, sounds good i'll do that and revert all these expectation changes